src/components/organisms/user-search/user-search.controller.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+import USER_SEARCH_QUERY from 'gql/user-search-query';
+import UserSearch from './user-search.view';
+
+const UserSearchController = () => {
```

Comment:
This component holds the business logic for the page:
- how to query github for results
- data remapping for easier component rendering downstream
- pagination functions
- state management

Having this here makes it easy to manipulate the view logic layer. Definitely want to get tests on this file before pulling it down.